java/bfg/Portfile

```diff
@@ -0,0 +1,57 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           java 1.0
+
+name                bfg
+version             1.13.0
+
+categories          java devel
+license             GPL-3.0
+maintainers         {@ra1nb0w irh.it:rainbow} openmaintainer
+platforms           darwin
+supported_archs     noarch
+
+description         Removes large or troublesome blobs like git-filter-branch does, but faster.
+long_description    The BFG is a simpler, faster alternative to git-filter-branch for cleansing \
+                    bad data out of your Git repository history: \n \
+                      - Removing Crazy Big Files \n \
+                      - Removing Passwords, Credentials & other Private data
+homepage            https://rtyley.github.io/bfg-repo-cleaner/
+
+master_sites        https://search.maven.org/remotecontent?filepath=com/madgag/${name}/${version}/
+distfiles           ${name}-${version}.jar
+distname            ${name}-${version}
+worksrcdir          ${name}-${version}
+
+checksums           rmd160  6f692650684b52f5f78801d57fddd2bc1d4d7595 \
+                    sha256  bf22bab9dd42d4682b490d6bc366afdad6c3da99f97521032d3be8ba7526c8ce \
+                    size    13465496
+
+depends_run         bin:java:kaffe
```

Comment:
Does the kaffe port even build anymore? I'm not sure that is the right way to specify the java requirement? Does the java portgroup handle this at all - what happens if you comment it out? I don't have time at the moment to give you the "right" answer but I'm hoping you (or someone knowledgeable of java in Macports) can research this a bit more.

Reply:
Otherwise this looks good to me!

Reply:
In fact, I am not sure on how declare java dependency. I copied that line from others ports on `java/`. Re-reading java group it should works if I add `java.fallback`, and removing `depends_run` of course, because it is empty by default. The following is the java group code

```tcl
    # Warn user if we couldn't find a likely JAVA_HOME
    if { ![file isdirectory $home_value]} {
        ui_warn "No value for java JAVA_HOME was automatically discovered"
    }

    # Add dependency if required
    if { ${java_version_not_found} && ${java.fallback} ne "" } {
        ui_debug "Adding dependency on JDK fallback ${java.fallback}"
        depends_lib-append port:${java.fallback}
    }
```
I changed the PR to respect this.

Reply:
That looks good now!